src/mpyl/steps/deploy/k8s/resources/dagster.py

```diff
@@ -18,10 +22,13 @@ class Constants:
 
 def to_user_code_values(
     project: Project,
+    release_name: str,
     name_suffix: str,
     run_properties: RunProperties,
     service_account_override: Optional[str],
     docker_config: DockerConfig,
+    sealed_secrets: Optional[List[V1EnvVar]] = None,
+    extra_manifests: Optional[List[CustomResourceDefinition]] = None,
```

Comment:
Shouldn't these just be `Lists` (without a default argument)? Then you don't need the `if sealed_secrets else []` later on. I think you pass `sealed_secrets_as_env_var` here anyway which is not optional. Maybe I missed something though

Reply:
no, it's a good comment! i will implement that